packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/utils/transformPermissionsData.js

```diff
@@ -0,0 +1,36 @@
+import { flatten } from 'lodash';
+
+const transformPermissionsData = (data) => {
```

Comment:
we don't need this

Reply:
you are right, we have removed all the transformPermissionsData file